### PR TITLE
New package: binfmt-support-2.1.5

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/wine/INSTALL [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# INSTALL
```
Comment (xtraeme) [inline srcpkgs/wine/INSTALL]: I'd like to see a xbps trigger rather than duplicating this into multiple packages.

See `srcpkgs/xbps-triggers/files`.

Comment (thypon) [inline srcpkgs/wine/INSTALL]: Only on wine or also for qemu. It's possible to copy binfmt files automatically with triggers?

Comment (xtraeme) [inline srcpkgs/wine/INSTALL]: The xbps trigger should run at post INSTALL time to register new entries for this package, and at post-remove time to unregister the pkg entries.

You'll have to introduce a new variable that accepts one argument or more, like:

```
binfmt="/usr/bin/qemu"
```

and then xbps-src will have to activate the binfmt trigger to be run at post-install or post-remove time.

Note this needs multiple changes to xbps-src and xbps-triggers.

Comment (thypon) [inline srcpkgs/wine/INSTALL]: I have no time to generalize also the qemu case. I'd need architecture checker helpers. The wine one shouldn't be a problem.

**Comment by thypon** [comment]
@xtraeme better? :-)

**Comment by xtraeme** [comment]
Looks good if it works as expected :+1:

**Comment by thypon** [comment]
I fixed two lines in xbps-trigger and added qemu-user-static binfmts. If it's ok could be merged right now.

**Pull request merged**